src/views/List.jsx

```diff
@@ -19,12 +20,20 @@ export function List({ listPath, data, listName, userIdFromPath }) {
 
 	const sortedItems = filteredItems.sort(comparePurchaseUrgency);
 
+	if (!currentUserId) {
+		return (
+			<div>
+				<p>Sign in to view your list items.</p>
+			</div>
+		);
+	}
+
 	if (!data.length) {
 		return (
 			<div>
 				{currentUserId && listName && <p>List name: {listName}</p>}
 				<label htmlFor="add-first-item">
-					There are no items in this list. Click this button to add your first
+					There are no items in this list. Click this link to add your first
 					items!
```

Comment:
This works as expected. However, I must say that as user I'd like to see this on the landing page which is currently the `Home` page. Have you considered displaying the message there as well? Or hiding the `List` and `Manage List` page until the user sings in?

Reply:
We can discuss this as, right not the Home page has Modals that pop up if a user interacts with it but isn't signed in.

Reply:
Yes, feel free to ignore this comment. It was just an observation/non-blocking comment.  This PR meets the AC. I’d focus on addressing the merge conflict and merging the branch.